src/framework/standard/parse/mod.rs

```diff
@@ -505,17 +511,40 @@ pub async fn command(
             Map::Prefixless(subgroups, commands) => {
                 is_prefixless = true;
 
+                fn command_name_if_recognised(res: &Result<Invoke, ParseError>) -> Option<&str> {
+                    match res {
+                        Ok(Invoke::Command {
+                            command, ..
+                        }) => Some(command.options.names.get(0).copied().unwrap_or("<unknown>")),
+                        Ok(Invoke::Help(name)) => Some(name), // is this correct?
```

Comment:
Help commands are handled above, from line 480. This is a useless match arm since `Invoke::Help` isn't conceived by either `handle_group` or `handle_command`, but I don't see harm in it. It can be kept, I suppose.